eng/testing/runsettings.targets

```diff
@@ -0,0 +1,40 @@
+<Project InitialTargets="GenerateRunSettingsFile">
+
+  <PropertyGroup>
+    <RunSettingsInputFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsInputFilePath>
+    <RunSettingsAppOutputFilePath>$(OutDir).runsettings</RunSettingsAppOutputFilePath>
+
+    <RunSettingsOutputFilePath>$(RunSettingsAppOutputFilePath)</RunSettingsOutputFilePath>
+
+    <ResultsFileName>$(MSBuildProjectName)_$(TargetFramework)_$(TargetPlatform)</ResultsFileName>
```

Comment:
Why do you need this? Do test projects in this repo not have a unique intermediate and output directory?

Reply:
This is because we use arcade's test result directory, so I was following arcade's output convention when running `build.cmd -test`. I wanted to produce the same results file if you run that vs `dotnet run` in order to not have different results file.

But to answer your question we do have different intermediate and bin output per project but arcade puts test results under `artifacts/TestResults/<Configuration>`